server/models/user-model.js

```diff
@@ -225,6 +225,10 @@ userSchema.methods.registerUser = async function () {
  * @property {string} companyName - company name of the user.
  * @property {string} password - password of the user.
  */
+async function getAllCompanyName(){
+ return(await User.distinct('companyName'));
+}
```

Comment:
Cool implementation. I didn't know about `distinct` operations.
Two things about this:

- The function is between the user model declaration and it's documentation, so now the documentation will point to your function instead of the user model.
- If you want to keep the same style as the rest of the model, you can make the function into a static method like this:

```javascript
userSchema.statics.getAllCompanyNames = async function() {
  return User.distinct('companyName');
}
```

Using `return await` outside of a `try-catch` block is redundant, and will show up as an error in some linters.

If you do that, you won't have to export he function separately, and you can use it by simply doing `User.getAllCompanyNames()`